### DISCUSSION TITLE
wws:// Rest API

### DISCUSSION THREAD

**Discussion by smurraystrata** [Budibase/budibase#9135] [Help] [answered]
Not sure if this should be a help or a feature request :). 

I am trying to make an API call to a system that uses wws:// instead of https://. When I put the URL (wws://IP:18443/rest/v1/alarm/ws) in and send the API call Budibase adds an HTTP:// to the beginning of the URL and gives me an error.

Query Error: request to http://wws//IP:18443/rest/v1/alarm/ws failed, reason: getaddrinfo EAI_AGAIN wws

This URL does work when I use curl or other methods. Any ideas on how to get Budibase to quit trying to be smart and adding the HTTP:// and removing the : in the wws://?

Thank you!

**Comment by melohagan** [comment, marked as answer]
Hey @jedman20 

We do not currently support Websockets or any other form of real-time pub/sub functionality.
